expirationd-scm-1.rockspec

```diff
@@ -8,7 +8,7 @@ description = {
     summary = "Expiration daemon for Tarantool",
     homepage = "https://github.com/tarantool/expirationd",
     license = "BSD2",
-    maintainer = "Eugine Blikh <bigbes@tarantool.org>"
+    maintainer = "Sergey Bronnikov <sergeyb@tarantool.org>"
```

Comment:
There is a maintainer field in `debian/control` as well.

Reply:
Updated too.